test/mpdecimal_test.exs

```diff
@@ -2,27 +2,79 @@ defmodule MPDecimalTest do
   use ExUnit.Case
   doctest MPDecimal
 
-  test "correct digit handling" do
-    result = MPDecimal.Nif.power("3.14159", "1.58732")
-    assert result == "6.153686306332765322891030633"
+  test "positive integer exponents" do
+    assert_exp(2, 2, "4.0")
```

Comment:
I grabbed most of these tests from the internal test suite we had written for the commission engine.